[FW][FIX] stock_currency_valuation: FIX SLV company context errors

fw-bot-adhoc opened the pull request:
Forward-Port-Of: ingadhoc/stock#727

maq-adhoc committed:
[FIX] stock_currency_valuation: FIX SLV compamy context errors

X-original-commit: 602defc167025da288f1abd6818a22cf27875c23

roboadhoc commented:
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/stock/pull/728).

fw-bot-adhoc commented:
@maq-adhoc @feg-adhoc cherrypicking of pull request ingadhoc/stock#727 failed.

stdout:
```
CONFLICT (modify/delete): stock_currency_valuation/views/product.xml deleted in HEAD and modified in de8f039f ([FIX] stock_currency_valuation: FIX SLV compamy context errors). Version de8f039f ([FIX] stock_currency_valuation: FIX SLV compamy context errors) of stock_currency_valuation/views/product.xml left in tree.
CONFLICT (modify/delete): stock_currency_valuation/models/stock_valuation_layer.py deleted in HEAD and modified in de8f039f ([FIX] stock_currency_valuation: FIX SLV compamy context errors). Version de8f039f ([FIX] stock_currency_valuation: FIX SLV compamy context errors) of stock_currency_valuation/models/stock_valuation_layer.py left in tree.
CONFLICT (modify/delete): stock_currency_valuation/models/stock_move.py deleted in HEAD and modified in de8f039f ([FIX] stock_currency_valuation: FIX SLV compamy context errors). Version de8f039f ([FIX] stock_currency_valuation: FIX SLV compamy context errors) of stock_currency_valuation/models/stock_move.py left in tree.
CONFLICT (modify/delete): stock_currency_valuation/__manifest__.py deleted in HEAD and modified in de8f039f ([FIX] stock_currency_valuation: FIX SLV compamy context errors). Version de8f039f ([FIX] stock_currency_valuation: FIX SLV compamy context errors) of stock_currency_valuation/__manifest__.py left in tree.

```

stderr:
```
14:04:47.996363 git.c:444               trace: built-in: git cherry-pick de8f039f1087b4daf2efebd3d3b18ddbc8fbdfa6
error: could not apply de8f039f... [FIX] stock_currency_valuation: FIX SLV compamy context errors
hint: after resolving the conflicts, mark the corrected paths
hint: with 'git add <paths>' or 'git rm <paths>'
hint: and commit the result with 'git commit'
----------
status:

```

Either perform the forward-port manually (and push to this branch, proceeding as usual) or close this PR (maybe?).

In the former case, you may want to edit this PR message as well.

More info at https://github.com/odoo/odoo/wiki/Mergebot#forward-port

rov-adhoc commented:
Se cierra por que no esta en 17
